aten/src/ATen/ScalarOps.h

```diff
@@ -13,14 +13,14 @@ inline at::Tensor scalar_to_tensor(Scalar s, const Device device = at::kCPU) {
   // This is the fast track we have for CPU scalar tensors.
   if (device == at::kCPU) {
     if (s.isFloatingPoint()) {
-      return at::native::scalar_tensor(s, at::device(at::kCPU).dtype(at::kDouble));
+      return at::native::scalar_tensor(s, at::kDouble, c10::nullopt, at::kCPU);
```

Comment:
nit: nicer to say `/* layout */ c10::nullopt`, since the non-TensorOptions calling convention has severe "optional" blindness.